docs/install_guides/installing-openvino-linux.md

```diff
@@ -33,6 +33,12 @@
   
   .. _Product Specifications: https://ark.intel.com/
 
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+
```

Comment:
We have Software tab for Windows install guide. I'm not sure how it's strict regarding GCC version, but in general it makes sense to add these requirements in order to align with software requirements defined in  the [Release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes.html)

@ilyachur Can you add also this tab for APT (Ubuntu 18 and 20) and YUM (RHEL8) install guides?

Reply:
@slyubimt @ilyachur 
Can we also add this to releases/2021/4 ? This is LTS release and users hit into such compiler compatibility issue: https://github.com/openvinotoolkit/openvino/issues/10929

Reply:
Fixed

Reply:
PR for 2021/4 https://github.com/openvinotoolkit/openvino/pull/11191